library/core/src/convert/num.rs

```diff
@@ -144,7 +144,7 @@ impl_from! { i16, isize, #[stable(feature = "lossless_iusize_conv", since = "1.2
 
 // CHERI proposes 256-bit “capabilities”. Unclear if this would be relevant to usize/isize.
```

Comment:
Current CHERI uses 128-bit capabilities.

Reply:
That paper does describe 256-bit capabilities, both in the old "unpublished" link and in the newer link. Is there a more current reference we could use?

Reply:
https://www.cl.cam.ac.uk/techreports/UCAM-CL-TR-951.pdf:

> [...] In our initial work on CHERI, we utilized an uncompressed capability
format in which each capability was 256 bits in size. This gave us significant flexibility to ex-
periment with capability contents and semantics for bounds checking and capability behaviors.
As our protection and software models matured, we turned our attention to the performance
implications of large capability sizes, including potentially substantial data-cache overhead
for pointer-intensive applications. We adapted an approach explored in fat-pointer research –
bounds compression exploiting redundancy between a pointer value and its bounds – to imple-
ment 128-bit compressed capabilities. [...]

Reply:
```suggestion
// CHERI proposes 128-bit “capabilities”. Unclear if this would be relevant to usize/isize.
```